osbs/utils.py

```diff
@@ -87,7 +87,7 @@ def write_file(self, name, content):
 
 
 class TarReader(object):
-    TarFile = namedtuple('TarFile', ['filename', 'fileobj'])
+    tar_file = namedtuple('tar_file', ['filename', 'fileobj'])
```

Comment:
Not sure I agree with this one. TarFile makes it clearer that it's a defined class/type, not an instance or method. If it really can't be a class member it could perhaps move to module scope, but that seems to give it more scope that required.